1-js/03-code-quality/03-comments/article.md

```diff
@@ -90,7 +90,7 @@ for(let t = 0; t < 3; t++) {
 // ...
 ```
 
-Then it might be a better variant to refactor it into functions like:
+फिर इसे करने के लिए एक बेहतर संस्करण हो सकता है functions का प्रयोग करना:
```

Comment:
This sentence could be improved like `फिर इसे functions में रिफैक्टर करने के लिए यह एक बेहतर संस्करण हो सकता है जैसे:`